src/components/blockquote.tsx

```diff
@@ -16,31 +17,37 @@ interface Props {
     format: Format;
 }
 
-const styles = (format: Format): SerializedStyles => css`
-    font-style: italic;
-    position: relative;
-    margin: ${remSpace[4]} 0 ${remSpace[9]} 0;
-    padding: 0 ${remSpace[6]};
+const styles = (format: Format): SerializedStyles => {
+    const { kicker } = getPillarStyles(format.pillar);
+    return css`
+        font-style: italic;
+        position: relative;
+        margin: ${remSpace[4]} 0 ${remSpace[9]} 0;
+        padding: 0 ${remSpace[5]};
+
+        svg {
```

Comment:
Could we use the same space formatting used in this file?

Reply:
👍